infra/envs/dev/main.tf

```diff
@@ -32,10 +32,36 @@ provider "aws" {
 //  preferred_maintenance_window = "${var.preferred_maintenance_window}"
 //}
 
+module "browser_site_cert" {
+  source = "github.com/chanzuckerberg/cztack//aws-acm-cert?ref=v0.29.0"
+
+  # the cert domain name
+  cert_domain_name = "browser.dev.single-cell.czi.technology"
```

Comment:
How was this domain decided on?

Reply:
d**ev.single-cell.czi.technology** domain already existed for us. I executively decided **browser**, because that's what we have been calling it.

Reply:
If we decided on a better name later we can change it, but we needed a domain with an SSL cert for https for auth0 to work.

Reply:
Got it! As long as it won't be a hassle to change it in the future, this sounds good to me.